features/feedback.feature

```diff
@@ -1,30 +1,48 @@
-    And I should be on the sign in page
+@javascript @no-seed
+Feature: A user can provide feedback and bug report
+  @stub_survey_monkey_success
+  Scenario: A user can successfully submit feedback
+    Given I visit "/"
+    When I click the link 'Provide feedback'
+    Then I should see a page title "Help us improve this service"
+    Then the page should be accessible skipping 'aria-allowed-attr'
 
-  Scenario: An advocate is unable to sign in and selects to submit feedback and is prompted for their email and ignores it
-    Given I have not signed in
-    When I click the link 'feedback'
-    Then I should see 'What is your email address?'
-    When I fill in the 'feedback' form
-    Then I expect ZendeskSender to receive a description without an email
-    And I see confirmation that my 'feedback' was received
-    And I should be on the sign in page
+    And I choose govuk radio 'Yes' for 'Were you able to complete the tasks you aimed to on Claim for crown court defence today?'
+    And I choose govuk radio 'Very satisfied' for 'How satisfied have you been in your experience of Claim for crown court defence today?'
+    And I fill in 'Tell us about your experience of using the service today' with 'This is great!'
+    And I click govuk checkbox 'Other (please specify)'
+    And I fill in 'Enter your comment' with 'Something Else'
+    And I click the button 'Send'
 
-  Scenario: An advocate is unable to sign in and selects to submit feedback and is prompted for their email and completes it
-    Given I have not signed in
-    When I click the link 'feedback'
-    Then I should see 'What is your email address?'
-    When I fill in the 'feedback' form with email of 'test@example.com'
-    Then I expect ZendeskSender to receive a description with an email
-    And I see confirmation that my 'feedback' was received
+    Then I see confirmation that my 'feedback' was received
     And I should be on the sign in page
+
+  @stub_survey_monkey_failure
+  Scenario: A user receives notification of a failure to submit feedback
+    Given I visit "/"
+    When I click the link 'Provide feedback'
+    Then I should see a page title "Help us improve this service"
+    Then the page should be accessible skipping 'aria-allowed-attr'
+
+    And I choose govuk radio 'Yes' for 'Were you able to complete the tasks you aimed to on Claim for crown court defence today?'
+    And I choose govuk radio 'Very satisfied' for 'How satisfied have you been in your experience of Claim for crown court defence today?'
+    And I fill in 'Tell us about your experience of using the service today' with 'This is great!'
+    And I click govuk checkbox 'Other (please specify)'
+    And I fill in 'Enter your comment' with 'Something Else'
+    And I click the button 'Send'
+
+    Then I see a warning that my feedback was not submitted successfully
+    And I should be on the feedback page
+
+# TODO: Introduce a new test case for bug report
```

Comment:
needs reimplementing

Reply:
@jsugarman This is a new test case. I couldn't implement it now as the zendesk doesn't work locally. We need to implement this test as part of the work where we split the bug report out to its own controller and model.

Reply:
I have created a ticket - CFP-281

Reply:
@imtiazAhmd we can just stub zendesk anyway

here is a [PR against this branch](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4302) to add the feature in. works locally